src/magic.cpp

```diff
@@ -617,6 +617,8 @@ std::string spell::duration_string() const
     if( has_flag( spell_flag::RANDOM_DURATION ) ) {
         return string_format( "%s - %s", moves_to_string( min_leveled_duration() ),
                               moves_to_string( type->max_duration ) );
+    } else if( has_flag( spell_flag::PERMANENT ) ) {
```

Comment:
note that permanent duration only happens at max spell level

Reply:
@KorGgenT I don't understand. This is meant for any spells with the `PERMANENT` flag, regardless of what level they are.

Reply:
I see what you mean now - spawned *items* are given the `ETHEREAL_ITEM` flag, unless they are (a) comestible, or (b) both `PERMANENT` and at maximum level.

https://github.com/CleverRaven/Cataclysm-DDA/blob/237b16ed2078e562d6ddad1fdaa1bf5b033a89d2/src/magic_spell_effect.cpp#L757-L760

Spawned *monsters* on the other hand need only have the `PERMANENT` flag to be permanent (spell level does not matter):

https://github.com/CleverRaven/Cataclysm-DDA/blob/237b16ed2078e562d6ddad1fdaa1bf5b033a89d2/src/magic_spell_effect.cpp#L884-L886

Assuming this is intended, I will include a check here for what type of spell it is (spawn item or spawn monster), and will include some tests for it.